packages/client/components/AnalyticsPage.tsx

```diff
@@ -61,8 +62,26 @@ const AnalyticsPage = () => {
       window.analytics = mockSnippet
     }
   }, [])
-  const [isSegmentLoaded] = useScript(`https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`, {crossOrigin: true})
+  const [isSegmentLoaded] = useScript(
+    `https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`,
+    {
+      crossOrigin: true
+    }
+  )
   const atmosphere = useAtmosphere()
+  useEffect(() => {
+    if (window.__ACTION__.logRocketEnabled) {
+      const email = window.localStorage.getItem(LocalStorageKey.EMAIL)
+      LogRocket.init(window.__ACTION__.logRocketId, {
+        release: __APP_VERSION__
+      })
+      if (email) {
```

Comment:
this hook only runs once, which means it's possible that `email` could be undefined if we haven't cached it yet.
may also want to include the identify call in https://github.com/ParabolInc/parabol/blob/b7259570dd3be40fc48cd02715fa9a422deb12c1/packages/client/utils/handleSuccessfulLogin.ts

Reply:
Makes sense! So it might not be cached when the user logs in but by including the identifier there and here, we should always have the email?

Reply:
yep!